File: POO/Ejercicio1/Persona.java
public class Persona {
    private String nombre;
    private int edad;
    private String ciudad;

    public Persona(String nombre, int edad, String ciudad) {
        this.nombre = nombre;
        this.edad = edad;
        this.ciudad = ciudad;
    }

    public String mostrarSaludo() {
        return String.format("HOLA MI NOMBRE ES %s Y VIVO EN LA CIUDAD DE %s", this.nombre, this.ciudad);
    }

    public String VerificarMayorEdad() {
        if (this.edad >= 18) {
            return this.nombre + " es mayor de edad";
        } else {
            return this.nombre + " no es mayor de edad";
        }
    }

    public static void main(String[] args) {
        Persona persona1 = new Persona("Maria", 26, "Cochabamba");
        Persona persona2 = new Persona("Jose", 15, "La Paz");
        Persona persona3 = new Persona("Silvia", 19, "Lima");

        System.out.println(persona1.mostrarSaludo());
        System.out.println(persona2.mostrarSaludo());
        System.out.println(persona3.mostrarSaludo());

        System.out.println(persona1.VerificarMayorEdad());
        System.out.println(persona2.VerificarMayorEdad());
        System.out.println(persona3.VerificarMayorEdad());
    }
}

File: POO/Ejercicio1/Persona.py
class Persona:
    def __init__(self, nombre, ciudad, edad):
        self.nombre = nombre
        self.ciudad = ciudad
        self.edad = edad

    def saludo(self):
        print(f"Hola, soy {self.nombre} y vivo en la ciudad de {self.ciudad}")

    def es_mayor(self):
        if self.edad > 18:
            print(f"{self.nombre} es mayor de edad tiene {self.edad} años")
        else:
            print(f"{self.nombre} no es mayor de edad tiene {self.edad} años")


# Parte principal (equivalente a main en Java)
if __name__ == "__main__":
    p1 = Persona("Juana", "Sucre", 25)
    p2 = Persona("Mery", "Cordova", 15)
    p3 = Persona("Pablo", "Montevideo", 24)

   
    p1.saludo()
    p2.saludo()
    p3.saludo()

 
    p1.es_mayor()
    p2.es_mayor()
    p3.es_mayor()